frappe/public/js/frappe/utils/utils.js

```diff
@@ -1528,5 +1528,9 @@ Object.assign(frappe.utils, {
 				return [doctype, filter, val[0], val[1], false];
 			});
 		}
+	},
+
+	load_video_player() {
+		return frappe.require("video_player.bundle.js");
```

Comment:
not sure about this, this does return a promise so developers can chain whatever they want to do after this 👀 

any better way?

Also I don't like the way bundle "imports" library into frappe namespace. Not sure if there's any better alternative.